bricks/widget/__brick__/{{name.snakeCase()}}.dart

```diff
@@ -1,7 +1,12 @@
 import 'package:flutter/material.dart';
 
 class {{name.pascalCase()}} extends StatelessWidget {
-  const {{name.pascalCase()}}({Key key}) : super(key: key);
+  const {{name.pascalCase()}}({
+    Key? key,
+    required this.child,
+  }) : super(key: key);
+
+  final Widget child;
```

Comment:
Thoughts on just returning a `const SizedBox()` instead for simplicity? I think most developers would end up deleting the `child` if we injected it via constructor like this.

Reply:
>  I think most developers would end up deleting the `child`

Good point.
As the `child` was already there in the `Container` (few lines below), I figured it needs to be there.

We could also just stick to empty `Container`. One reason to support this idea is that it is what Flutter plugin for VS Code generates when you type "stless". But of course, I don't mind the `const SizedBox()`.

Reply:
If it sounds good to you then can we just adjust to return a `const SizedBox()` since it simplifies the template and reduces the amount of code developers need to change when they generate their own widgets? Thanks again for bringing this up!

Reply:
Sure! I pushed the changes.

Btw, I started exploring mason today (I don't know what took me so long :disappointed:) and... wow... this is amazing!
I submitted my application for brickhub today and described an idea I have so I guess you'll be hearing from me some more in the future :slightly_smiling_face:

Reply:
Thanks so much! That's awesome, looking forward to collaborating 🚀